src/component/NavBar.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { NavLink } from 'react-router-dom';
+import planet from './assets/planet.png';
+import './NavBar.css';
+
+const NavBar = () => (
+  <header>
+    <nav className="nav-container">
+      <div className="logo-container">
+        <img src={planet} alt="Logo" className="planet-logo" />
+        <h1>space Traveller&apos;s Hub</h1>
+      </div>
+      <ul className="nav-items">
+        <li className="li">
+          <NavLink to="/" id="navlink" className={({ isActive }) => (isActive ? 'active' : '')}>Rocket&apos;s</NavLink>
+        </li>
+        <li className="li">
+          <NavLink to="/Mission" id="navlink" className={({ isActive }) => (isActive ? 'active' : '')}>Mission&apos;s</NavLink>
+        </li>
+        <li className="li">
+          <NavLink to="/MyProfile" id="navlink" className={({ isActive }) => (isActive ? 'active' : '')}>My Profile</NavLink>
+        </li>
+      </ul>
+    </nav>
+  </header>
```

Comment:
- [ ] One of the requirements is to ensure that when a navigation link is clicked, it should be underlined. Although you haveimplemented it using the react-router-dom Navlink component, you are sharing the same ID for all navigation links. Remember that in CSS, IDs ought to be unique. Since you want to share the same style declarations for all the navigation links, you can use a class instead Kindly implement this change. 😎 💻 

#### Hint:
 Your jsx should look like this:
 
```jsx
<ul className="nav-items">
  <li className="li">
    <NavLink to="/" className={`${({ isActive }) => (isActive ? 'active' : '')} navlink`}>Rocket&apos;s</NavLink>
  </li>
  <li className="li">
    <NavLink to="/Mission" className={`${({ isActive }) => (isActive ? 'active' : '')} navlink`}>Mission&apos;s</NavLink>
  </li>
  <li className="li">
    <NavLink to="/MyProfile" className={`${({ isActive }) => (isActive ? 'active' : '')} navlink`}>My Profile</NavLink>
  </li>
</ul>
```

Then in your CSS, you can do this:

```CSS
.navlink {
  list-style: none;
  text-decoration: none;
  color: rgb(6, 167, 172);
}

a.active {
  color: rgb(128, 6, 0);
  text-decoration: underline;
}

```